cluster/src/main/java/com/hartwig/pipeline/transfer/google/GoogleArchiver.java

```diff
@@ -21,7 +21,7 @@ public GoogleArchiver(Arguments arguments) {
     public void transfer(SomaticRunMetadata metadata) {
         LOGGER.info("Starting transfer from [{}] to GCP bucket [{}]", arguments.outputBucket(), arguments.archiveBucket());
         try {
-            GSUtil.configure(false, 12);
+            GSUtil.configure(true, 12);
```

Comment:
Do we want verbose for production code? May be quite useful for debugging, may be just noise.